compile.sh

```diff
@@ -5,3 +5,4 @@ RUSTFLAGS="-Zlocation-detail=none" cargo +nightly build --manifest-path rust/Car
 kotlinc -include-runtime kotlin/code.kt -d kotlin/code.jar
 #kotlinc-native -include-runtime kotlin/code.kt -d kotlin/code
 dart compile exe dart/code.dart -o dart/code --target-os=macos
+dotnet publish ./csharp -c Release -r linux-x64 -o csharp/bin --self-contained
```

Comment:
https://github.com/bddicken/languages/blob/1c9592da35099d7f1e5cee3c77b7e498fcf19bc8/compile.sh#L8
https://github.com/bddicken/languages/blob/ae5cfceae8d7813291d5dc91d7232a75e0256d53/compile.sh#L8

I'm not familiar with dotnet but folks in other PRs that add C# were able to make the command platform-agnostic

Reply:
Since we run it locally in the same environment, yes, it is indeed possible. Thank you for noticing this.